profiles/package.mask

```diff
@@ -65,12 +65,15 @@ dev-java/ant-xz
 dev-java/gnu-jaf
 dev-java/failureaccess
 
+# Oz Tiram <oz.tiram@gmail.com> (2024-03-10)
+# plette in version 1.0.0 is broken and pipenv is still
+# not using it. Bug #925611
+=dev-python/plette-1.0.0
 # Michał Górny <mgorny@gentoo.org> (2024-03-10)
 # Packages that still require <dev-python/cython-3 and do not promise
 # a fix anytime soon.
 #
 # dev-python/python-efl: econnman as revdep, bug #925692.
-# dev-python/pipenv: via <pydantic-2, + dparse, bug #925698, bug #925553.
```

Comment:
This seems to be the wrong commit for this.

Reply:
I moved unmask to a separate commit. I hope I understood you. Maybe it should be with the same commit as adding 2023.12.1? I can squash those commits if necessary.